File: benchmarks/list-100/main.go
package main

import "fmt"
import "time"

func main() {
	var before = time.Now().UnixNano()

	var list = make([]int, 0)
	for i := 0; i < 100; i++ {
		list = append(list, i)
	}

	var after = time.Now().UnixNano()

	fmt.Println(after - before)
}

File: benchmarks/list-100/main.rs
use std::time::{SystemTime};

fn main() {
    let before = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();

    let mut list = vec![];

    for i in 0..100 {
        list.push(i);
    }

    let after = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH).unwrap();
    println!("{}", after.as_nanos() - before.as_nanos());
}